src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -719,18 +741,27 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                     if (result.HasFix)
                     {
                         Logger.Log(FunctionId.SuggestedActions_HasSuggestedActionsAsync);
-                        return true;
+                        if (result.Diagnostic.Severity == DiagnosticSeverity.Error)
+                        {
+                            // Show the error lightbulb if we know there's an error fix availalbe
+                            return PredefinedSuggestedActionCategoryNames.ErrorFix;
```

Comment:
+1 especially since this is for an A/B test, if we can't return the correct icon almost 100% of the time, I think that we should consider dropping the error icon altogether.

As it is, could this code be refactored to wait in parallel for the first fix at each severity level?